doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -295,7 +295,285 @@ Your output should look like the following:
+    
+To install the operator onto the cluster:
+
+-  Download the operator installer YAML using the following command:
+
+   ::
+
+       wget https://raw.githubusercontent.com/aws/amazon-sagemaker-operator-for-k8s/master/release/rolebased/namespaced/operator.yaml
+
+- Update the installer YAML to place the resources into your specified namespace using the following command:
+
+   ::
+   
+       sed -i -e 's/PLACEHOLDER-NAMESPACE/<YOUR NAMESPACE>/g' operator.yaml
+
+-  Edit the \ ``installer.yaml`` file to
+   place resources into your \ ``eks.amazonaws.com/role-arn``. Replace the ARN here with
+   the Amazon Resource Name (ARN) for the OIDC-based role you’ve created. 
+
+-  Use the following command to deploy the cluster:  
+
+   ::
+
+       kubectl apply -f installer.yaml
+
+Deploy the Operator to Your Namespace Using Helm Charts
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+There are two parts needed to deploy an operator within the scope of a namespace. The first is the set of CRDs that are installed at a cluster level. These resource definitions only need to be installed once per Kubernetes cluster. The second part is the operator permissions and deployment itself. When using helm charts you will have to first create the namespace using kubectl.
+
+
+Clone the Helm installer directory using the following command:
+
+::
+
+    git clone https://github.com/aws/amazon-sagemaker-operator-for-k8s.git
+
+Navigate to the
+``amazon-sagemaker-operator-for-k8s/hack/charts/installer/namespaced`` folder. Edit
+the \ ``rolebased/values.yaml`` file, which includes high-level parameters for the
+Chart. Replace the role ARN here with the Amazon Resource Name (ARN) for the OIDC-based role you’ve
+created. 
+
+Install the Helm Chart using the following command:
+
+::
+
+    helm install crds crd_chart/
+
+
+Create the required namespace and install the operator using the following command:
+
+::
+
+    kubectl create namespace <namespace>
+    helm install --n <namespace> op operator_chart/
+
+
+.. warning::
+    If you decide to install the operator into a namespace other than the one specified above,
+    you will need to adjust the namespace defined in the IAM role ``trust.json`` file to match.
+
+After a moment, the chart will be installed with a randomly generated
+name. Verify that the installation succeeded by running the following
+command:
+
+::
+
+    helm ls
+
+Your output should look like the following:
+
+::
+
+    NAME                    NAMESPACE                       REVISION        UPDATED                                 STATUS          CHART                           APP VERSION
+    sagemaker-operator      sagemaker-k8s-operator-system   1               2019-11-20 23:14:59.6777082 +0000 UTC   deployed        sagemaker-k8s-operator-0.1.0
+
+
+Verify the operator deployment to your namespace
```

Comment:
This section needs changes, I will update the quip doc.

Reply:
Actually only the namespace needs to change to "my-namespace" in line 601 and line 605